contracts/traits/errors/diamond.rs

```diff
@@ -0,0 +1,19 @@
+use super::OwnableError;
+
+/// The Diamond error type. Contract will throw one of this errors.
+#[derive(Debug, PartialEq, Eq, scale::Encode, scale::Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+pub enum DiamondError {
+    OwnableError(OwnableError),
+    FunctionAlreadyExists,
```

Comment:
That error can be more informative if it contains the old code hash of the owner of the selector.